### DISCUSSION TITLE
NAV adókódok

### DISCUSSION THREAD

**Discussion by Enkiducsa** [nav-gov-hu/eVAT#1] [Általános]
Nagyon sűrítve, mert tucatnyi jó dolog még felmerülhet, - számomra - a legizgalmasabb a NAV áfa kódok. Miért? Mert a tervezet szerint ezt kell kikényszeríteni. amit rendeleti/törvénymódosítás útján lehet megvalósítani. Valójában jobb lenne ezt nem kényszernek, hanem lehetőségnek tekinteni, és lehet is annak tekinteni. Egyszerűsítve miről is van szó? A NAV egységes adórendszerét oly módon tartja fenn, hogy a beérkező adatok alapján alkalmazott eljárásokkal ehhez adókódokat rendel, amit a benyújtott bevallások soraiba rendezve az adatokat, jelennek meg - egyszerűsítés, de ha nagyon tévedek akkor jó lenne, ha ezt az oldalt teljesen átlátható módon ki lehetne elemezni. 
A másik oldal az adatszolgáltató. Amint a tervezetben is megjelent, vélelmezhető, hogy minden ERP/Könyvelő program rendelkezik saját adókódokkal, amennyiben 65-ös bevallás xml-t készít. E kettő összhangjáról van szó. A kényszer tehát az, hogy legyen olyan matematikai eljárás, ami tetszőleges, és akár dinamikusan változó NAV kódrendszer, és a szolgáltató kódrendszere között megállapítja a közös halmazt. Ilyen elméleti matematikai modell ugyanis létezik. Mi lehet ennek az előnye? Az adatszolgáltató kaphat egy választ, hogy a saját áfa kód rendszerét az algoritmus egyöntetűnek tekinti-e vagy sem (A=B). Amennyiben nem akkor is megadja a közös halmazt, és mutatja azt a részhalmazt, ami kívül esik az adatszolgáltató adókód rendszeréből, mert nem illeszkedik. Ezt kell feloldani, mert nem minden adat szolgáltatónak nem kell leképezni a teljes NAV adókód rendszert, csak nem lehet olyan elem, amit a NAV kód rendszerével nem alkot közös halmazt, másként az adatszolgáltató rendszer minden áfakódja a konverziót követően közös halmazban kell legyen NAV áfakód rendszerével - kérdem, jól értem?
Ehhez mihamarabb meg kellene ismerni a NAV kódrendszert, mielőtt a sok szolgáltató nekiugrik a fejlesztésnek.

**Comment by renced42** [comment]
Kedves @Enkiducsa az Adókód lista publikálásra került. A https://github.com/nav-gov-hu/eVAT/tree/master/docs/standard_adokod linken elérhető.

**Comment by pahitamas** [comment]
A link nem működik nálam. Csak nálam?

**Comment by NTCA-developer** [comment]
@pahitamas mindjárt javítom. Itt lakik egyébként: https://github.com/nav-gov-hu/eVAT/tree/master/docs/standard_adokod
